eq-author-api/tests/utils/graphql.js

```diff
@@ -136,25 +131,8 @@ const createAnswerMutation = `
 const createOtherMutation = `
   mutation CreateOther($input: CreateOtherInput!) {
```

Comment:
I guess we should rename this as well. `CreateAdditional`?

Reply:
Should be able to remove it altogether and fold it into create option hopefully.